testing/resize-observer-stub.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {pushIfNotExist, removeItem} from '../src/utils/array';
+
+/**
+ * @param {!Object} sandbox
+ * @param {!Window} window
+ * @return {!ResizeObservers}
+ */
+export function installResizeObserverStub(sandbox, win) {
```

Comment:
this file is a great idea! really need one for InOb too!

Reply:
Yeah. I thought we'll try this pattern for these observers.